### PR TITLE
use ES6 syntax

### PR DESCRIPTION
**Pull request by App-Developer8866**
Following changes:

- Use ES6 module syntax
- Show time

### CODE REVIEW

**Review by Gambit142 — COMMENTED** [review]
Hi @App-Developer8866 👋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Kindly include a Future Features section in your README.md file. In this section, you can outline 1-3 features that can be implemented in the future. 😎 
- The link to your MIT LICENSE is broken. Kindly fix it. 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: main.js [hunk 1]
```diff
@@ -1,84 +1,49 @@
...
-  bookc.removeBook(id);
+// remove book
+const removeBook = (id) => {
+  const books = JSON.parse(localStorage.getItem('data')) === null ? [] : JSON.parse(localStorage.getItem('data'));
+  if (books.length > 0) {
+    const filteredBooks = books?.filter((i) => i?.id !== id);
+    localStorage.setItem('data', JSON.stringify(filteredBooks));
+    window.location.reload();
+  }
 };
```
Comment (DuaneDave) [inline main.js]: As part of the project requirements, you should use es6 import keywords but I haven't seen the usage of that and it is coming from the revert of the PR. Could you fix this to stay consistent with the milestone requirements?


Also, this is what your app currently looks like, I do not see the date displayed as expected. Kindly implement this to make the project more like the wireframe.

errApp

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @App-Developer8866 👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Readme looks great and professional
✔️ Correct usage of GitHub flow
✔️ Very professional PR summary
✔️ Books can be added and removed


## Required Changes ♻️
- According to the project requirements, you should have a file namely, index.html, index.js, and index.css but you currently have style.css and main.js and index.html. Please rename this file to stay in sync with the milestone [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md#:~:text=Create%20the%20main%20index.html%20file.%20Link%20your%20index.js%20and%20index.css%20files%20there.)
- The requirements also states that there must exist a folder called modules, which i noticed you had but when you reverted your PR, it was gone. Kindly add this to stick with the project [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md#:~:text=Create%20a%20directory%20called%20modules.)

THIS A SCREENSHOT OF YOUR FILE STRUCTURE

modules

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,122 +1,184 @@
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- Author -->
+## 👥 Authors <a name="authors"></a>
 
 👤 **Author**
 
 - GitHub: [@App-Developer8866](https://github.com/App-Developer8866/)
 - Twitter: [@MMuttrafRasool](https://twitter.com/MMuttrafRasool)
 - LinkedIn: [Muhammad Muttraf Rasool](https://www.linkedin.com/in/muhammad-muttraf-rasool-421819202)
```
Comment (amon-cofie) [inline README.md:134]: [OPTIONAL] Great work on the readme file. To make it even more professional, I think you can replace ```Author``` with your name

**Review by amon-cofie — APPROVED** [review]
Hi @App-Developer8866 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Readme looks great and professional
✔️ Correct usage of GitHub flow
✔️ Books can be added and removed
✔️ Changes well implemented
✔️ Good commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**